### DISCUSSION TITLE
Nvidia gpu

### DISCUSSION THREAD

**Discussion by allanc1** [blakeblackshear/frigate#1321] [General]
Hi
I know that integration of nvidia gpus into frigate is on your agenda but wonder what your timetable might be? Coral cards are difficult to come by ATM the use of gpus adds a significant degree of flexibility. Asmirnou has submitted a solution and wonder if this is a workable interim solution once updated to Ubuntu 20.04 and using tensorRT 8.0?

**Comment by Honest-Objections** [comment]
+1 for this; they're likely not available in certain countries too.

Absolutely love this project though, fantastic work!
